style.css

```diff
@@ -553,69 +555,301 @@ textarea {
+  }
+
+  .aboutme-container {
+    width: 100vw;
+    margin-top: 112px;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+
+  .skills-container {
+    display: flex;
+    margin-top: 52px;
+  }
+
+  .project2 {
+    border: 1px solid #c1c7d0;
+    margin: 30px;
+    padding: 20px 1px;
+    height: 352px;
+    width: 369px;
+    background-color: #ebf0ee;
     border-radius: 10px;
-    padding-left: 20px;
-    padding-right: 20px;
-    padding-top: 5px;
-    padding-bottom: 5px;
-    margin-top: 10px;
   }
 
-  .card-header {
-    position: absolute;
-    padding: 13px;
-    right: 0;
-    z-index: 500;
+  .form-section {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    width: 76%;
+    margin: 80px auto 110px;
+    background-image: url(./images/illustration-2-contact-form-desktop@2x.png);
+    background-repeat: no-repeat;
+    background-position: bottom left;
+  }
+
+  .form-description {
+    width: 40%;
   }
-}
 
-@media (min-width: 768px) {
-  .display-centered {
+  .contact-form {
+    width: 55%;
+  }
+
+  input,
+  textarea {
+    width: 100%;
+  }
+
+  .new-background {
+    background-image: url(./images/illustration-2-contact-form-desktop.png);
+    background-repeat: no-repeat;
+    background-position: bottom left;
+    margin-left: 119px;
+  }
+
+  .work-section {
+    width: 100vw;
+    min-height: 100vh;
     display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
   }
 
-  .mobile-buttons {
-    display: none;
+  .about-text {
+    width: 920px;
+    font-size: 20px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0 auto;
+  }
+
+  .li-lang {
+    border: 5px solid #fff;
+    background-color: #fff;
+    color: #36b37e;
+    font-size: 12px;
+    font-family: 'Inter', sans-serif;
+    border-radius: 8px;
+  }
+
+  .aboutme section {
+    display: flex;
+    background-image: url(/images/illustration-about\ me-1-desktop@2x.jpg);
+    background-repeat: no-repeat;
+    background-position: top right;
+  }
+
+  .new section {
+    display: flex;
+  }
+
+  .touch {
+    font-family: 'Inter', sans-serif;
+    font-style: bold;
+    font-size: 30px;
+    text-align: center;
+    align-items: center;
+    font-weight: 700;
+    line-height: 52px;
+    height: 256px;
+    color: #172b4d;
+  }
+
+  .button {
+    height: 50px;
+    display: flex;
+    justify-content: flex-start;
+  }
+
+  .contact {
+    display: flex;
+    background-image: url(./images/illustration-1-contact-form-desktop.png), url(./images/contact\ form-illustratoin-3-desktop.png);
+    background-repeat: no-repeat;
+    background-position: bottom left, bottom right;
+  }
+
+  .btn {
+    background-color: #36b37f;
+    color: white;
+    width: 159px;
+    height: 48px;
+    margin: 20px 0;
+    font-size: 17px;
+    border: none;
+    border-radius: 5px;
+  }
+
+  .btn:hover {
+    background-color: #36b37f;
+  }
```

Comment:
- [ ] Good job trying to include the button interaction 👍🏽 . Kindly note that the hover state button interaction has a box shadow that you didn't add. Kindly add it to all your buttons 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/213889647-fcb0b192-c81a-460d-91fd-b8068f7b1f06.png)